### DISCUSSION TITLE
Private data size used to benchmark end-to-end OCR

### DISCUSSION THREAD

**Discussion by BasemElbarashy** [mindee/doctr#821] [Ideas]
Here https://mindee.github.io/doctr/using_models.html
you provided great benchmarking on different models vs AWS and google OCR using private data of invoices, resumes ... etc
I understand it is not possible to share them at the moment but Could you please provide 
1- the size of each category ? 
2- or at least How many invoices are being used in this evaluation?

**Comment by fg-mindee** [comment]
Hi @BasemElbarashy :wave: 

Thanks for the suggestion, that's an excellent idea!
I'll see how we can fit descriptive stats into the documentation (number of images, split by document categories, etc.)
